css/global-cart.css

```
@@ -0,0 +1,209 @@
+    user-select: none;
+    
+    /* Typography-forward design */
+    font-family: var(--font-display);
+    
+    /* Initial state - hidden */
+    opacity: 0;
+    transform: translateY(-20px) scale(0.9);
+    transition: all var(--transition-base);
+    
+    /* Hover states */
+    transition: all 0.2s ease;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

transition is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)